data/foss2cyber.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Ishan Velle",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Front-end Web Developer | Freelancer | Designer | Blogger"
+  "avatar": "https://github.com/foss2cyber.png"
```

Comment:
In the latest version you do not need to include `avatar` as it will be automatically fetched from your profile. Please do remove it and let me know if you face any issue.
the updated docs are available at https://linkfree.eddiehub.io/docs/quickstart

Reply:
Thanks for the info, mate!

Reply:
Bro, Do remove it, so that the JSON check will successfully pass.